scripts/lis_hyperv_platform/main.ps1

```diff
@@ -270,11 +275,17 @@ function Main {
             throw "Creating the LISA VM failed."
         }
 
-        $idRSAPriv = Get-Lisa
+        Write-Host "Getting LISA code..."
+        Get-LisaCode
+        Invoke-WebRequest -Uri $PuttyBinaries -OutFile "PuttyBinaries.zip"
+        if ($LastExitCode) {
+            throw "Failed to download Putty binaries."
+        }
+        Expand-Archive ".\PuttyBinaries.zip" ".\lis-test\WS2012R2\lisa\bin"
```

Comment:
there are some issues if we hardcode the lisa path, we need to add all the lis-test paths references to a variable